enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -44,12 +44,13 @@ import (
 )
 
 var (
-	Runtime     = flag.String("executor.oci.runtime", "", "OCI runtime")
-	runtimeRoot = flag.String("executor.oci.runtime_root", "", "Root directory for storage of container state (see <runtime> --help for default)")
-	pidsLimit   = flag.Int64("executor.oci.pids_limit", 2048, "PID limit for OCI runtime. Set to -1 for unlimited PIDs.")
-	cpuLimit    = flag.Int("executor.oci.cpu_limit", 0, "Hard limit for CPU resources, expressed as CPU count. Default (0) is no limit.")
-	dns         = flag.String("executor.oci.dns", "8.8.8.8", "Specifies a custom DNS server for use inside OCI containers. If set to the empty string, mount /etc/resolv.conf from the host.")
-	netPoolSize = flag.Int("executor.oci.network_pool_size", 0, "Limit on the number of networks to be reused between containers. Setting to 0 disables pooling. Setting to -1 uses the recommended default.")
+	Runtime          = flag.String("executor.oci.runtime", "", "OCI runtime")
+	runtimeRoot      = flag.String("executor.oci.runtime_root", "", "Root directory for storage of container state (see <runtime> --help for default)")
+	pidsLimit        = flag.Int64("executor.oci.pids_limit", 2048, "PID limit for OCI runtime. Set to -1 for unlimited PIDs.")
+	cpuLimit         = flag.Int("executor.oci.cpu_limit", 0, "Hard limit for CPU resources, expressed as CPU count. Default (0) is no limit.")
+	cpuRelativeLimit = flag.Int("executor.oci.cpu_relative_limit", -1, "CPU hard limit added to the task size, expressed as CPU count. If set to -1, only use cpu_limit as a hard limit.")
```

Comment:
Should this be millicpu or is 1 cpu (1000 milliCPU the min overhead?)

Could we call this executor.oci.task_buffer_millicpu or executor.oci.task_overhead_millicpu to make it clear that it's an amount of CPU to be added to the task size hard limit?

Reply:
Changed to milli-CPU and went with `executor.oci.task_millicpu_headroom` - `buffer` is a bit too overloaded imo and `overhead` sounds like some fixed cost that we're trying to offset